src/modules/workspaces/repositories/workspaces.repository.spec.ts

```diff
@@ -69,6 +70,25 @@ describe('SequelizeWorkspaceRepository', () => {
     });
   });
 
+  describe('bulkUpdateInvitesKeysAndUsers', () => {
+    it('When invites are being updated, then it should update each invite with the correct invitedUser and encryptionKey', async () => {
+      const invites = [newWorkspaceInvite(), newWorkspaceInvite()];
+
+      await repository.bulkUpdateInvitesKeysAndUsers(invites);
+
+      invites.forEach((invite, index) => {
```

Comment:
Although this could seem like a great idea, is better to hardcode what we expect to receive rather than doing dynamic checks that imply some logic, as that opens the door to introducing a bug on the test itself.